src/protocols/rdp/keyboard.c

```diff
@@ -533,6 +533,9 @@ static const guac_rdp_keysym_desc* guac_rdp_keyboard_send_defined_key(guac_rdp_k
     }
 
     /* Fire actual key event for target key */
+    guac_client_log(keyboard->client, GUAC_LOG_TRACE,
```

Comment:
should this logging line be after the `guac_rdp_send_key_event` ?
In case it fails?